python/lsst/ts/observatory/control/auxtel/latiss.py

```diff
@@ -266,6 +266,29 @@ async def setup_atspec(self, filter=None, grating=None, linear_stage=None):
             async with self.cmd_lock:
                 return await asyncio.gather(*setup_coroutines)
 
+    async def get_setup(self):
+        """Get the current filter, grating and stage position
+
+        Returns
+        -------
+        filter name : ``evt_reportedFilterPosition.name``
```

Comment:
These names should be like variable names, e.g.`filter_name` also, the return type should be the actual return type so `string` in this case and I think `float` for the linear stage.

Reply:
note that the names aren't the *actual* names of the variables that are returned... which I think it was you mean?

Reply:
yes, I noticed... but I mean, they should be valid variable names. If you don't want to document variable names you can document the type only, e.g.:

```
Returns
-------
int
    Description of anonymous integer return value.
```

https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
code returned pieces of objects, so rather than putting those pieces in variables and then returning them i just followed the example above.